objectivec/DevTools/full_mac_build.sh

```diff
@@ -315,15 +296,15 @@ if [[ "${DO_XCODE_TVOS_TESTS}" == "yes" ]] ; then
   )
   case "${XCODE_VERSION}" in
     [6-9].* )
-      echo "ERROR: Xcode 10.0 or higher is required to build the test suite." 1>&2
+      echo "ERROR: Xcode 10.2 or higher is required." 1>&2
       exit 11
       ;;
     10.* | 11.* | 12.*)
```

Comment:
How sensitive are we to 10.2... do we need to check the 10.*?

Reply:
This script is mainly just use for CI and anyone working on the library (me), so I'm good with keeping it simple at the moment.